packages/sdks/src/blocks/video/video.lite.tsx

```diff
@@ -45,11 +89,10 @@ export default function Video(props: VideoProps) {
               }
             : null),
         }}
-        src={props.video || 'no-src'}
         poster={props.posterImage}
         class="builder-video"
       >
-        {!props.lazyLoad && <source type="video/mp4" src={props.video} />}
+        <source type="video/mp4" {...props.lazyLoad ? { 'data-src': props.video } : { src: props.video }} />
```

Comment:
props.lazyLoad is a boolean right? Did you mean:
```suggestion
        <source type="video/mp4" {...(props.lazyLoad ? { 'data-src': props.video } : { src: props.video })} />
```